coderedcms/models/wagtailsettings_models.py

```diff
@@ -226,6 +226,16 @@ class Meta:
         verbose_name=_('Track button clicks'),
         help_text=_('Track all button clicks using Google Analytics event tracking. Event tracking details can be specified in each button’s advanced settings options.'),  # noqa
     )
+    below_head_scripts = models.TextField(
+        blank=True,
+        null=True,
+        verbose_name=_('Add tracking scripts between the <head> tags.'),
```

Comment:
A bit long for name, might work better as help_text. Please also double check that this renders correctly in the wagtail admin (e.g. `<head>` shows up properly and not as an actual HTML tag)

verbose_name= `<head> tracking scripts`

Ditto for body_scripts

Reply:
done. it looks like this: 
![tracking_analytics_dashboard](https://user-images.githubusercontent.com/69739605/114089239-e7154280-9883-11eb-85cb-beb123084d0c.png)